.env.example

```
@@ -20,6 +20,14 @@ SERVICE_PORT_EXT=8000
 # Always use the POSTGRES_DB name with _test
 POSTGRES_DB_TESTING=app_test
 
+# Telegram Bot
+TELEGRAM_BOT_USERNAME=padelpals-franco-test
```

```suggestion
TELEGRAM_BOT_USERNAME=
```
Creo que esta variable de entorno conviene dejarla vacía para que los que prueben se den cuenta que lo tienen que llenar en la validación de las .env
Muchas veces se saca el `.example` del `.env.example` y se fijan solo las faltantes